vtex/toolbelt
# Refact/plugin deps

#### What is the purpose of this pull request?
Transform `deps` command into plugin

#### What problem is this solving?

#### How should this be manually tested?
Run all deps subcommands and see if the behavior is as expected

`vtex-test deps diff`
`vtex-test deps update`
`vtex-test deps list`

#### Screenshots or example usage
![Screen Shot 2020-08-10 at 21 17 52](https://user-images.githubusercontent.com/19983991/89843406-f5715a00-db4e-11ea-8e17-e8e40dcddd7c.png)
![Screen Shot 2020-08-10 at 21 18 11](https://user-images.githubusercontent.com/19983991/89843419-002bef00-db4f-11ea-9e58-c5d8e7e6ecbe.png)
![Screen Shot 2020-08-10 at 21 18 31](https://user-images.githubusercontent.com/19983991/89843427-0c17b100-db4f-11ea-8fc4-b4fa1742450f.png)

#### Types of changes

#### Chores checklist
